lib/game.rb

```diff
@@ -29,46 +31,48 @@ def set_options
       @human_player.marker = @marker
     end
 
+    def generate_markers
+      markers = []
+
+      @players.each do |player|
+        markers << player.marker
+      end
+
+      markers
+    end
+
     def run_game_loop
-      until is_game_over?
+      markers = generate_markers
+
+      until TicTacToe::is_game_over?(@board, markers)
         if is_computer_the_current_player?
```

Comment:
Refer to the comment below about the player move logic. Ideally, the game should be able to ask the current player for its move without needing to know (or care) which player it is.